cadquery/occ_impl/shapes.py

```diff
@@ -1451,9 +1453,41 @@ def sweep_multi(cls, profiles, path, makeSolid=True, isFrenet=False):
         if makeSolid:
             builder.MakeSolid()
 
-
         return cls(builder.Shape())
 
+    def dprism(self, basis, profiles, depth=None, taper=0, thruAll=True,
+              additive=True):
+        """
+        Make a prismatic feature (additive or subtractive)
+
+        :param basis: face to perfrom the operation on
+        :param profiles: list of profiles
+        :param depth: depth of the cut or extrusion
+        :param thruAll: cut thruAll
+        :return: a Solid object
+        """
+
+        sorted_profiles = sortWiresByBuildOrder(profiles)
+        shape = self.wrapped
+        basis = basis.wrapped
+        for p in sorted_profiles:
+            face = Face.makeFromWires(p[0],p[1:])
+            feat = BRepFeat_MakeDPrism(shape,
+                                       face.wrapped,
+                                       basis,
+                                       #basis.normalAt().toDir(),
```

Comment:
Not sure if this comment was left in intentionally.

Reply:
Yes, will remove that.